docs/meeting-notes/foam-core-2020-07-11.md

```diff
@@ -0,0 +1,125 @@
+# Foam Core 2020-07-11
+
+Present: @jevakallio, @riccardoferretti
+
+### Tests
+
+- How do we know this approach works?
+  - Supports renaming
+  - Supports searching with (attribute-x)
+  - Find dead links
+
+### Getting started
+
+- Land work to master
+  - Create a foam-core package
+  -
+
+### Open questions
+
+- How should writing to files work
+  - What if affected notes have unsaved changes
+
+### Graph methods
+
+- get all
+- search by
+  - tag
+  - free text
+  - [[todo]]: how do vs code search editors work? are they pluggable? what do they need?
+- find dead links
+  - for linters
+- serialize/toJSON (for visualizers)
+- subscribe to changes
+
+### Node methods
+
+- rename
```

Comment:
This would probably rename both the node and all links to it? Or maybe that's in second phase, and in the first phase these would just be reported as invalid links. How about the other way around when refactor action (rename) would be applied to a link?

Reply:
It would have to rename node and all links. If we are to support `[[Page Title]]` links, there's also a special case here where the rename may have to not only regenerate the link reference definitions, but replace the link text inside the document itself.

```suggestion
- rename node and all links to that node
```